src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -460,3 +527,389 @@ def _transform_optional_constants(graph: torch.Graph):
             const_input.node().moveBefore(node)
             const_input.node().copyMetadata(node)
             node.output().replaceAllUsesWith(const_input)
+
+class TorchFXPythonDecoder (Decoder):
```

Comment:
It is better to maintain two different decoders in separate files. Could we separate them?

Reply:
Sure Sergey. Separating them into fx_decoder and ts_decoder.